src/website/views/components/mobile-navigation/_examples.pug

```diff
@@ -0,0 +1,317 @@
+h2 Examples
+h3 Base
+.example.-mb--3(style='overflow:hidden;')
+  .-position--relative.-z--0(style="height:30rem;overflow:hidden;")
+    .-p--3
+      each i in [1, 2, 3]
+        .-text.-mb--1 Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus eu dignissim nisi, gravida pharetra elit. Etiam eu urna orci. Nulla et lorem eleifend, ultrices massa id, molestie urna. Nulla nec quam in turpis fermentum dictum vitae ac nibh. Suspendisse lacus nisi, sollicitudin in commodo quis, euismod id enim. Donec semper nunc et tellus convallis, tristique varius turpis gravida. Quisque hendrerit magna ac bibendum molestie. Nullam scelerisque libero vitae lorem dignissim ullamcorper. Integer mollis auctor enim vel molestie. Etiam id vestibulum augue, vitae dapibus quam. Nunc tincidunt aliquet lacus nec malesuada. Donec ultricies augue non lorem eleifend, eget ullamcorper lacus elementum. Donec condimentum enim nec justo auctor, nec bibendum ipsum dapibus.
+    .chi-backdrop.-position--absolute
+      .chi-backdrop__wrapper
+        #drawer-11.chi-drawer.-left.-animated.-active.chi-mobile-nav.-position--absolute(style=`background-color: #242526;`)
+          .chi-divider.-colorbar
+          .chi-drawer__header.chi-mobile-nav__header
+            .chi-dropdown
+              button.chi-button.-flat.-light.chi-dropdown__trigger.-animate.-no-hover(id="button-site-menu2") Cloud Application Manager
+              .chi-dropdown__menu.-inverse.-list.-w--lg
```

Comment:
We'll need to stretch the application drop down menu width to the full width of the drawer and adjust the top value to render further down at the base of chi-drawer__header to match the mockup. You could apply a `-b--transparent` to hide the buttons white border on active.